build_all/make_sdk.cmd

```diff
@@ -21,20 +21,25 @@ rem // The location of the Arduino PAL directory relative to this file
 set Arduino_pal_path=%arduino_repo_root%pal\
 set AzureIoTSDKs_path=%arduino_repo_root%sdk\
 
-
 set AzureIoTHub_path=%Libraries_path%\AzureIoTHub\
 set AzureIoTProtocolHTTP_path=%Libraries_path%\AzureIoTProtocol_HTTP\
 set AzureIoTProtocolMQTT_path=%Libraries_path%\AzureIoTProtocol_MQTT\
-set AzureIoTProtocolAMQP_path=%Libraries_path%\AzureIoTProtocol_AMQP\
+rem set AzureIoTProtocolAMQP_path=%Libraries_path%\AzureIoTProtocol_AMQP\
```

Comment:
if this isn't being used it should probably be deleted

Reply:
Done.